projectDocs/translating/crowdin.md

```diff
@@ -1,45 +1,91 @@
-# Translating using Crowdin
+# Translating with Crowdin
 
-Crowdin is used to translate the main NVDA interface and user documentation.
+Crowdin is used to manage the translations of the main NVDA interface and user documentation.
 NVDA's Crowdin project: <https://crowdin.com/project/nvda>.
 
-This document covers setting up a Crowdin account, connecting it with PoEdit, and translating the main interface and user documentation using Crowdin and PoEdit.
+This document covers setting up a Crowdin account, and translating the main interface and user documentation using either the Crowdin web interface or Poedit / nvdaL10nUtil.
 
-## Setup
-
-### Create Crowdin account
+## Creating a Crowdin account
 
 1. Create a [Crowdin account](https://accounts.crowdin.com/register?continue=%2Fproject%2Fnvda).
 1. Message the [translators mailing list](https://groups.io/g/nvda-translations) or <info@nvaccess.org> to request being added as a translator.
 Please include your Crowdin username and the languages you wish to translate.
 
-### Setup PoEdit
+## Translation workflows
+
+There are 2 common workflows for translating with Crowdin:
+
+1. Translating strings directly via Crowdin's web interface. Or
+1. Downloading a translation file from Crowdin, translating with Poedit, and uploading the file back to Crowdin.
+
+### Translating via Crowdin's web interface
+
+For instructions on how to translate strings using Crowdin's web interface, please read [Crowdin's documentation for translators](https://support.crowdin.com/for-translators/).
+
+Note: If you are a screen reader user, you may find Crowdin's web interface inefficient or impossible to use.
+Therefore you may choose to use the Poedit workflow instead.
+
+#### Translation Reviews / Approvals
+
+Due to accessibility issues and to be able to support the alternative workflow using Poedit, the translation approvals feature has been disabled in the NVDA project on Crowdin.
+This means that all new strings are essentially auto approved.
+However, to maintain quality control, only translators specifically added to the project can add or change strings.
+
+### Translating with Poedit
+
+Poedit is a desktop application which is commonly used to translate files.
+It is fairly accessible and is used by many blind and vision impaired translators.
+
+The workflow for translating with Poedit involves:
+1. Downloading the translation file using NVDA's l10n utility
+1. Opening the translation file in Poedit, translating one or more strings, and saving the file.
+1. Uploading the translation file back to Crowdin.
+
+Warning: Do not download / upload translation files via Crowdin's web interface, nor use Poedit's Crowdin cloud translation feature, as these methods have been found to corrupt xliff files.
```

Comment:
Are you in contact with Crowdin for this?
Is this corruption bug expected to be fixed soon?

Reply:
I have had detailed email threads with both Poedit and Crowdin about this. Neither seem to be willing to address the bugs. Poedit is representing empty translations with `<target />`, which is probably not necessary. But much worse, Crowdin is treating that `<target/>` as the literal translation. I.e. in the xliff you get back from Crowdin you get `<target>&lt;target/&gt;>`. I cannot see how this is at all correct, but Crowdin is unwilling to fix it so far.